src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -86,6 +86,12 @@ object WorkspaceService {
   case object ListAllActiveSubmissions extends WorkspaceServiceMessage
   case class AdminAbortSubmission(workspaceNamespace: String, workspaceName: String, submissionId: String) extends WorkspaceServiceMessage
 
+  case class CreateGroup(groupRef: RawlsGroupRef) extends WorkspaceServiceMessage
```

Comment:
This stuff should not be in WorkspaceService. WorkspaceService is already too big and this stuff has nothing to do with workspaces. I can see it belonging in UserService.